library/ssl_tls.c

```diff
@@ -109,6 +109,33 @@ static void ssl_update_in_pointers( mbedtls_ssl_context *ssl,
 
 #if defined(MBEDTLS_SSL_PROTO_DTLS)
 
+/* Top-level Connection ID API */
+
+int mbedtls_ssl_set_cid( mbedtls_ssl_context *ssl,
```

Comment:
Please add a big fat warning to the documentation of the function that so far it does nothing (and always returns success). We want this branch to be mergeable, so that people who want to start coding against that API (as the stub functions aim to allow) can do it from the main branch (they might need other features from the main branch and shouldn't have to to there own merges), but I don't feel comfortable merging a stub implementation without it being documented.

Reply:
Yes. The PR was originally intended to be merged only when the implementation is complete, but as we discussed, this is no longer the case, and I agree with your remark in that case.

Reply:
Done.